game/state/gamestate.h

```diff
@@ -107,6 +109,7 @@ class GameState : public std::enable_shared_from_this<GameState>
 
 	int baseIndex = 1;
 	int difficulty = 0;
+	bool fundingTerminated = false;
```

Comment:
Any time you add something to the GameState (or any child classes) you need to add it to the gamestate_serialize.xml, as otherwise this value won't persist over save/load